### PR TITLE
Update Glossary

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
Add Lurking Wife Mode and update acronyms (like the rest of the document)

### CODE REVIEW

File: docs/glossary/README.md [hunk 1]
```diff
@@ -39,13 +39,20 @@ Read more: [What fee should I select?](/FAQ/FAQ-UseWasabi.md#what-fee-should-i-s
 :::
 
 :::details
-### Hardware Wallet
+### Fungibility
+
+Fungibility is a (missing) property of a bitcoin (or satoshis), which should make them indistinguishable from each other.
```
Comment (MaxHillebrand) [inline docs/glossary/README.md]: satoshis are fungible - utxos are not.

```suggestion
Fungibility is a property of a Bitcoin utxo which are indistinguishable from each other.
```

File: docs/glossary/README.md [hunk 2]
```diff
@@ -39,13 +39,20 @@ Read more: [What fee should I select?](/FAQ/FAQ-UseWasabi.md#what-fee-should-i-s
...
+Read more: [What fee should I select?](/why-wasabi/TransactionSurveillanceCompanies.md#attempt-to-destroy-fungibility)
```
Comment (MaxHillebrand) [inline docs/glossary/README.md]:
```suggestion
Read more: [Transaction surveillance companies attempting to destroy fungibility](/why-wasabi/TransactionSurveillanceCompanies.md#attempt-to-destroy-fungibility)
```

File: docs/glossary/README.md [hunk 3]
```diff
@@ -129,6 +142,13 @@ It's the quantity of equal value outputs of one CoinJoin transaction.
 Read more: [What is the anonimity set?](/FAQ/FAQ-UseWasabi.md#what-is-the-anonymity-set)
 :::
 
+:::details
+### Bloom Filter
+
+A filter used primarily by SPV clients to request only matching transactions and merkle blocks from full nodes.
```
Comment (MaxHillebrand) [inline docs/glossary/README.md]:
```suggestion
A filter used primarily by SPV clients to request only block headers and merkle proofs of a given transaction from full nodes.
```

File: docs/glossary/README.md [hunk 4]
```diff
@@ -176,13 +196,27 @@ Read more: [What is the dust threshold](/FAQ/FAQ-UseWasabi.html#what-is-the-dust
...
 
 KYC (Know Your Customer) is the process of a business being forced to identify and verify the identity of its clients, and to share this information with a government.
 The term is also used to refer to the bank regulation which governs these activities.
 Read more: [AML/KYC Information](/why-wasabi/TransactionSurveillanceCompanies.md#aml-kyc-information)
 :::
 
+:::details
+### Lurking Wife Mode (LWM)
+
+Lurking Wife Mode is a Wasabi feature that hides sensitive and critical information on the wallet itself.
```
Comment (MaxHillebrand) [inline docs/glossary/README.md]:
```suggestion
Lurking Wife Mode is a Wasabi feature that hides sensitive and critical information on the wallet itself, which is useful for screenshots.
```

File: docs/glossary/README.md [hunk 5]
```diff
@@ -176,13 +196,27 @@ Read more: [What is the dust threshold](/FAQ/FAQ-UseWasabi.html#what-is-the-dust
...
+Read more: [Lurking Wife Mode](/using-wasabi/LurkingWifeMode.md)
+:::
+
+:::details
+### Pay to EndPoint (P2EP)
+
+Pay to EndPoint simplifies the sending of bitcoins with more security and privacy than Pay to IP (P2IP).
```
Comment (MaxHillebrand) [inline docs/glossary/README.md]:
```suggestion
Pay to EndPoint is sending of bitcoins where the receiver adds one of his own coins as input for a two party CoinJoin.
```

I've never heard of P2IP?

File: docs/glossary/README.md [hunk 6]
```diff
@@ -198,7 +232,16 @@ Read more: [Wasabi's Solution](/why-wasabi/BitcoinPrivacy.md#wasabi-s-solution-4
 :::
 
 :::details
-### Tor
+### Taint
+
+Taint is equivalent to the 'trail' that a Bitcoin transaction leaves during the course of its journey.
+The taint analysis of a Bitcoin transaction evaluates the association between an address and the previous addresses involved in transactions.
```
Comment (MaxHillebrand) [inline docs/glossary/README.md]:
```suggestion
The taint analysis of a Bitcoin transaction evaluates the association between an address involved in the chain of transactions.
```

File: docs/glossary/README.md [hunk 7]
```diff
@@ -198,7 +232,16 @@ Read more: [Wasabi's Solution](/why-wasabi/BitcoinPrivacy.md#wasabi-s-solution-4
...
+The higher the number of taint, the stronger the connection between the two addresses.
```
Comment (MaxHillebrand) [inline docs/glossary/README.md]:
```suggestion

```

File: docs/glossary/README.md [hunk 8]
```diff
@@ -39,13 +39,20 @@ Read more: [What fee should I select?](/FAQ/FAQ-UseWasabi.md#what-fee-should-i-s
 :::
 
 :::details
-### Hardware Wallet
+### Fungibility
+
+Fungibility is a property of a Bitcoin utxo which are indistinguishable from each other.
```
Comment (yahiheb) [inline docs/glossary/README.md]: @MaxHillebrand it contradicts this:
`However, an unspent transaction output, a bitcoin, is a chunk of money and is not fungible` https://docs.wasabiwallet.io/using-wasabi/Receive.html#the-importance-of-labeling

Comment (MaxHillebrand) [reply docs/glossary/README.md]: Well, the goal of Bitcoin privacy is to have fungibility in UTXOs [sats are already fungible].

So a non-fungible UTXO is tied to your KYC data.
a fungible UTXO is CoinJoined [or other tools] so it's equal among peers [anonset]

Comment (yahiheb) [inline docs/glossary/README.md]:
```suggestion
Fungibility is a property of Bitcoin UTXOs that are indistinguishable from each other.
```

File: docs/glossary/README.md [hunk 9]
```diff
@@ -39,13 +39,20 @@ Read more: [What fee should I select?](/FAQ/FAQ-UseWasabi.md#what-fee-should-i-s
 :::
 
 :::details
-### Hardware Wallet
+### Fungibility
+
+Fungibility is a property of Bitcoin UTXOs that are indistinguishable from each other.
```
Comment (MaxHillebrand) [inline docs/glossary/README.md]:
```suggestion
Fungibility is a desirable property of Bitcoin UTXOs that are indistinguishable from each other.
```

**Comment by RiccardoMasutti** [comment]
@MaxHillebrand yes should be ready

**Pull request merged**